src/documents/tutorial-userInterfaceOptions/UserInterfaceOptions.md

```diff
@@ -214,6 +206,19 @@ Here's the complete example from start to finish. This example assumes the Infus
 
         <nav class="flc-toc-tocContainer"> </nav>
 
+        <!-- Initialize the UI Options Javascript -->
+        <script type="text/javascript">
+            $(document).ready(function () {
+                fluid.uiOptions.prefsEditor(".flc-prefsEditor-separatedPanel", {
+                    tocTemplate: "lib/infusion/src/components/tableOfContents/html/TableOfContents.html",
+                    terms: {
+                        templatePrefix: "lib/infusion/src/framework/preferences/html",
+                        messagePrefix: "lib/infusion/src/framework/preferences/messages"
+                    }
+                });
+            })
+        </script>
```

Comment:
We had talked about the initialization script. With it using a `$(document).ready` it could appear anywhere in the document really, including the `<head>` I believe. In the `<head>` would mean it would start to interpret sooner, but it's so small that it probably doesn't matter. Do you have thoughts on where the best place is?